syntax = "proto3";

package sas;

message MyRequest {
}

message MyResponse {
  string data = 1;
}

service MyService {
  rpc MyMethod(MyRequest) returns (MyResponse) {}
}


service MyClient {
  rpc MyClientMethod(MyRequest) returns (MyResponse) {}
}

service MyClient2 {
  rpc MyClientMethod2(MyRequest) returns (MyResponse) {}
}
